Sample Solutions/Viva Insights Query Analysis Automation/Notebooks/vivaInsights_person.ipynb

```diff
@@ -88,7 +69,8 @@
         "from pyspark.sql.types import DateType\r\n",
         "\r\n",
         "#Reading meeting csv file from storage account\r\n",
-        "personDf = spark.read.csv(inputFilePath.format(extractionFS, StorageAccountName, PipelineId, PersonQueryDatasetFolder), header = 'true', inferSchema= 'true')\r\n",
+        "inputFilePath = 'abfss://{0}@{1}.dfs.core.windows.net/{2}/raw/{3}/*.txt'.format(VivaInsightsDataFileSystem, StorageAccountName, PipelineId, MeetingQueryDatasetFolder)\r\n",
+        "personDf = spark.read.csv(inputFilePath, header = 'true', inferSchema= 'true')\r\n",
         "\r\n",
         "# Dataframe prep\r\n",
         "personDf = personDf.withColumn(\"Date\",personDf['Date'].cast(DateType()))\r\n",
```

Comment:
It's for the next line. not sure how to add another comment to a past review. Would SecondaryEmployeeId need to be enclosed in quotes?

"personDf = personDf.withColumnRenamed(SecondaryEmployeeId,\"EmployeeId\")\r\n",

Reply:
I misunderstood and this is a parameter. closing this comment